data/hai/unfettered side missions.txt

```diff
@@ -160,267 +96,413 @@ mission "Hai Legacy 1"
-			`	"As such, most will say that you have no right to use that ship. Luckily for you, I do not believe in any of that. What matters to me is that this ship is very durable, given the time it obviously spent in the water. It also has ancient technology that would be very valuable to us."`
-			`	They raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this relic."`
+			`As you land, you realize your ship is the center of the spaceport crowd's attention, probably due to the ancient design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual based on your experiences with the Unfettered. "Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of.`
+			`	"As such, most will say that you need to prove to us you have a right to use that ship. Lucky for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology, as well as Korath technology we have a right to salvage for."`
+			`	Her tone suddenly changes as she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
 			choice
 				`	"What do I get if I win?"`
 					goto win
 				`	"Can't I just sell it to you?"`
-			`	"No, because in the eyes of my brethren that would mean I did not deserve this ship."`
+				`	"I bought this ship. It's mine and I shall do what I please with it." (Leave.)`
+					goto headingout
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
 			label win
-			`	"You will win the right to access our worlds for just a meagre daily fee of 500 credits. In order to keep our respect, and to not lose out on this priceless artifact, this Ancient Ladybug may not be destroyed or sold. You will not be allowed to land so easily as you just did, should you refuse.`
-			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you, will you take to the skies, or are you too afraid?"`
+			`	"You will win the right to access our worlds for just a meager daily fee of 600 credits. In order to keep our respect, this Anomalocaris may not be destroyed or sold, until you prove yourself another way. This is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here without paying a fee.`
+			`	"Beware, for there are no official rules, other than the fact that you may only use this one ship. So, will you take to the skies, or are you too afraid?"`
```

Comment:
Quick question: Does this filter out mission escorts as well? ~~Totally not about to cheat through battles like this.~~

Reply:
they wont help you :mad_hurl:
just added marked to all ships - if bounties can have it then duellists surely can